libtiledbsoma/src/tiledbsoma/tiledbsoma

```diff
@@ -49,9 +49,6 @@
 #include "soma/column_buffer.h"
 #include "soma/soma_array.h"
 #include "soma/soma_collection.h"
-#include "soma/soma_column.h"
```

Comment:
I don't understand why these lines are being deleted on this PR

Reply:
They are intended to be used internally (within `SOMAArray` and subclasses) so exporting them seemed unnecessary. In any case if the usage pattern changes, we can export them again or do it preemptively now, I have no opinion on that.